### PR TITLE
Update reference.mdx

### PR DESCRIPTION
**Pull request by Dziurwa14**

### CODE REVIEW

File: pages/reference.mdx [hunk 1]
```diff
@@ -24,9 +24,10 @@ Discord exposes different versions of our API. You can specify which version to
 
 | Version | Status                           | Default |
 | ------- | -------------------------------- | ------- |
+| 10      | Available                        |         |
 | 9       | Available                        |         |
-| 8       | Available                        |         |
-| 7       | Doesn't look like anything to me |         |
+| 8       | Deprecated                       |         |
+| 7       | Deprecated                       |         |
 | 6       | Deprecated                       | ✓       |
```
Comment (splatterxl) [inline pages/reference.mdx:31]: Shouldn't these be discontinued?

Comment (dolfies) [reply pages/reference.mdx:31]: They haven't discontinued them yet for reasons:tm:

**Comment by configari** [comment]
Shitting rn

File: pages/reference.mdx [hunk 2]
```diff
@@ -289,23 +290,34 @@ Discord uses ids and hashes to render images in the client. These hashes can be
...
+| Guild Banner                     | /banners/[guild_id](/resources/guild#guild-object)/[guild_banner](/resources/guild#guild-object).png                                                                                                                        | PNG, JPEG, WebP           |
+| Guild Template Icon              | /template/[template_code](/resources/guild-template#guild-template-structure)/icons/[template_icon](/resources/guild-template#guild-template-structure).png                                                                 | PNG, JPEG, WebP           |
+| Guild Home Header                | /guilds/[guild_id](/resources/guild#guild-object)/home-headers/[guild_home_header](/resources/guild#guild-object).png                                                                                                       | PNG, JPEG, WebP           |
+| Guild Eventnt                    | /guild-events/event_id/home-headers/[event_icon].png                                                                                                                                                                        | PNG, JPEG, WebP           |
```
Comment (dolfies) [inline pages/reference.mdx:303]:
```suggestion
| Guild Event                      | /guild-events/event_id/home-headers/[event_icon].png                                                                                                                                                                        | PNG, JPEG, WebP           |
```

File: pages/reference.mdx [hunk 3]
```diff
@@ -289,23 +290,34 @@ Discord uses ids and hashes to render images in the client. These hashes can be
...
+| User Avatar                      | /avatars/[user_id](/resources/user#user-object)/[user_avatar](/resources/user#user-object).png \*                                                                                                                           | PNG, JPEG, WebP, GIF      |
+| User Avatar Decoration           | /avatars/[user_id](/resources/user#user-object)/avatar-decorations/[user_avatar_decoration](/resources/user#user-object).png                                                                                                | PNG                       |
+| User Guild Avatar                | /guilds/[guild_id](/resources/guild#guild-object)/users/[user_id](/resources/user#user-object)/avatars/[user_avatar](/resources/user#user_object).png \*                                                                    | PNG, JPEG, WebP, GIF      |
+| User Guild Banner                | /guilds/[guild_id](/resources/guild#guild-object)/users/[user_id](/resources/user#user-object)/banners/[user_banner](/resources/user#user_object).png \*                                                                    | PNG, JPEG, WebP, GIF      |
```
Comment (dolfies) [inline pages/reference.mdx:310]:
```suggestion
| Guild Member Avatar              | /guilds/[guild_id](/resources/guild#guild-object)/users/[user_id](/resources/user#user-object)/avatars/[user_avatar](/resources/user#user_object).png \*                                                                    | PNG, JPEG, WebP, GIF      |
| Guild Member Banner              | /guilds/[guild_id](/resources/guild#guild-object)/users/[user_id](/resources/user#user-object)/banners/[user_banner](/resources/user#user_object).png \*                                                                    | PNG, JPEG, WebP, GIF      |
```
And move these two to be under guild stuff

**Review by dolfies — CHANGES_REQUESTED** [review]

**Review by dolfies — APPROVED** [review]

**Pull request merged**